db/migrate/20170421193150_add_embedded_methods_to_miq_ae_method.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmbeddedMethodsToMiqAeMethod < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_ae_methods, :embedded_methods, :string, :array => true
```

Comment:
@bdunne It contains the method names and its possible that we could have lengths longer than 255 if they have a handful of methods defined. I have changed it to a text field.